src/games/strategy/net/DesktopUtilityBrowserLauncher.java

```diff
@@ -79,18 +83,19 @@ public static void openFile(final File file) {
    *        A web address (URL) of a web page (ex: "http://www.google.com/")
    */
   public static void openURL(final String url) {
-    try { // attempt to use Desktop library from JDK 1.6+
-      final Class<?> d = Class.forName("java.awt.Desktop");
-      d.getDeclaredMethod("browse", URI.class)
-          .invoke(d.getDeclaredMethod("getDesktop").invoke(null), URI.create(url));
-      // above code mimicks: java.awt.Desktop.getDesktop().browse()
-    } catch (final Exception ignore) { // library not available or failed
+    boolean failed = true;// TODO check if the failed thing is necessary
+    if (Desktop.isDesktopSupported()) {
+      try {
+        Desktop.getDesktop().browse(URI.create(url));
+        failed = false;
+      } catch (IOException e) {
+        ClientLogger.logError(e);
+      }
+    }
+    if (failed) {
       final String osName = System.getProperty("os.name");
       try {
-        if (osName != null && osName.startsWith("Mac OS")) {
-          Class.forName("com.apple.eio.FileManager").getDeclaredMethod("openURL", String.class)
```

Comment:
OpenURL method simply throws a RuntimeException. At least thats what the java decompiler tells me